google/cloud/pubsub/internal/message_callback.h

```diff
@@ -36,6 +37,9 @@ class MessageCallback {
   struct MessageAndHandler {
     pubsub::Message message;
     std::unique_ptr<pubsub::ExactlyOnceAckHandler::Impl> ack_handler;
+    std::string ack_id;
+    // A single subscribe span, if it exists.
```

Comment:
Add: "This value is not known initially. The `TracingBatchCallback` will supply it."